scipy/stats/stats.py

```diff
@@ -6056,7 +6147,7 @@ def ks_2samp(data1, data2, alternative='two-sided', mode='auto'):
 
     Parameters
     ----------
-    data1, data2 : sequence of 1-D ndarrays
+    x, y : sequence of 1-D ndarrays
```

Comment:
same question as above: can we rename these parameters?

Reply:
I'll change them back in `stats.ks_2samp`.
But that raises an interesting question on `kstest`.
The signature of `kstest` changed:
```
def kstest(rvs, cdf, args=(), N=20, alternative='two-sided', mode='approx')`
-->
def kstest(data1, data2, args=(), N=20, alternative='two-sided', mode='auto')
```

The default `mode` should not be an issue.  But the name change of `rvs, cdf` -> `data1, data2` might be.  The changed names reflect the enhanced nature of the function.  It supports the old use cases of `kstest`, which handled one sample testing.  It also supports two sample testing via `kstest(rvs1, rvs2)`, in which case `rvs2` is definitely not a cdf of any kind.

@rgommers @tylerjereddy Is it permitted to change the names of positional arguments of a public API function?
I did a quick Google search for "kstest(rvs=..." and found a couple of examples specifying `rvs` by keyword, so my assumption is that changing the argument names at this time is not the thing to do.  Just document very clearly that `cdf` can be more than just a `cdf`. Is that correct?